app/controllers/tasks_controller.rb

```diff
@@ -43,13 +44,13 @@ def update_tasks
   def destroy
     @task = current_user.tasks.find(params[:id])
     @task.destroy
-    redirect_to :action => 'list'
+    render json: @task
```

Comment:
this necessary?

Reply:
ya, the redirect throws errors.
we do have the option:
1) render with task a remove single task from front end store.
2) render all tasks from back end after delete